clang-tools-extra/docs/clang-tidy/checks/bugprone/tagged-union-member-count.rst

```diff
@@ -9,6 +9,9 @@ different from the number of data members inside the union.
 A struct or a class is considered to be a tagged union if it has
 exactly one union data member and exactly one enum data member and
 any number of other data members that are neither unions or enums.
+The union and enum data members that are from system header files or
+the ``std`` namespace are not considered to make up the tagged union part
+of a user-defined tagged union type.
```

Comment:
Maybe `Union and enum data members that are from system header files or
the ``std`` namespace are not considered to be the union member or tagging enum member respectively of a user-defined tagged union type.`